README.md

```diff
@@ -1,4 +1,6 @@
-# ZK Eigen Trust
+# ZK EigenTrust - Deprecated
```

Comment:
How about removing the word `Deprecated`?
I think that EigenTrust project is not "deprecated", but just no activity.